collect_app/src/main/java/org/odk/collect/android/formentry/FormEntryUseCases.kt

```diff
@@ -241,4 +247,21 @@ object FormEntryUseCases {
                 )
         }
     }
+
+    private fun addDeprecatedId(formController: FormController) {
```

Comment:
This can be moved to javarosa later.

Reply:
This is a good idea, but it seems it may not be feasible. In contrast to handling forms with entities and dynamic preload, in this case we can't determine when something should happen. With forms that use entities or dynamic preloads, you can read the data from the form and perform the necessary actions.

However, with editable forms, you can only check if the form is editable. To determine whether it’s an edit that requires action, we would likely need to track instance IDs in our database and compare them with the ID parsed from the form. If it's an edit, we could then add the deprecatedID.

If you strongly support this idea, we can file a separate issue. However, I’m not sure it makes sense to introduce changes to our database schema for this kind of improvement.

Reply:
Ah you are right, thanks!